.gitignore

```diff
@@ -67,6 +67,7 @@ cgroup-network
 !cgroup-network/
 
 ebpf.plugin
+collectors/ebpf.plugin/reset_netdata_trace.sh
```

Comment:
You included the file to `.gitignore` but forgot to delete it from the repository.

Reply:
I removed it, thank you to call attention for this.